src/Definition/Common.php

```diff
@@ -58,6 +72,22 @@ public function isNullable()
     /**
      * @return string
      */
+    public function getDefault()
```

Comment:
I was wondering if we wanted to do that at this point.  the way it is right now will not change the way sapi works, adding `default` and `nullable` means supporting them in sapi.